python/lsst/afw/display/interface.py

```diff
@@ -80,7 +80,7 @@ def _makeDisplayImpl(display, backend, *args, **kwargs):
         try:
             _disp = importlib.import_module(dt, package="lsst.display")
             break
-        except ImportError as e:
+        except (ImportError, SystemError) as e:
```

Comment:
That is what seems to be triggered from `import_module` if `lsst.display` does not exist.